plugins/charamel-ws/src/main/java/de/dfki/vsm/xtension/charamelWs/charamelWsExecutor.java

```diff
@@ -150,18 +150,24 @@ public void launch() {
 
         mLogger.message(sceneflowVar);
 
-        app = Javalin.create(config -> {
-            config.server(() -> {
-                Server server = new Server();
-                ServerConnector sslConnector = null;
-                sslConnector = new ServerConnector(server, getSslContextFactory());
-                sslConnector.setPort(wss_port);
-                ServerConnector connector = new ServerConnector(server);
-                connector.setPort(ws_port);
-                server.setConnectors(new Connector[]{sslConnector, connector});
-                return server;
-            });
-        }).start();
+        if (mPathToCertificate != null) {
+            app = Javalin.create(config -> {
+                config.server(() -> {
+                    Server server = new Server();
+                    ServerConnector sslConnector = null;
+                    sslConnector = new ServerConnector(server, getSslContextFactory());
+                    sslConnector.setPort(wss_port);
+                    ServerConnector connector = new ServerConnector(server);
+                    connector.setPort(ws_port);
+                    server.setConnectors(new Connector[]{sslConnector, connector});
+                    return server;
+                });
+            }).start();
+        }
+        else {
+            app = Javalin.create(config -> config.enforceSsl = true).start(ws_port);
```

Comment:
enforceSsl is just an automatic redirect from http to https. Should this be enabled when not running https?